<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/sy.jpg</normaloff>:/sy.jpg</iconset>
  </property>
  <widget class="QPushButton" name="but_usr_manage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/yh.ico</normaloff>:/yh.ico</iconset>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="but_book_manage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书管理</string>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/book.ico</normaloff>:/book.ico</iconset>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>0</y>
     <width>841</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>781</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="managepage_usr_input">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="managepage_usr_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_search_row">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查询用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_add_row_usr">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>增加用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_remove_row_usr">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>430</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>删除该行用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_ent_mod">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_add_data">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>430</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导入数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_echo_data">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>430</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导出数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_to_pwd">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>密码修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_reflash">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>搜索:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_book_search">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="but_bookcharge">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_book_delete">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_bsch">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QTableView" name="book_tableView">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>751</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="but_bookadd">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="but_borrow_manage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>借阅管理</string>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/jieyue.ico</normaloff>:/jieyue.ico</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
